rinex/src/navigation/health/gal.rs

```diff
@@ -1,88 +1,84 @@
-//! GAL Sv Health specifications
-use bitflags::bitflags;
+//! GAL SV Health specifications
 
-/// GAL Sv Health indication
-#[derive(Debug, Clone, PartialEq, PartialOrd)]
+/// GAL INAV & FNAV SV Health indication.
```

Comment:
In RINEX v3 and v4 the Galileo health definition is the same for both INAV and FNAV

Though there is another field "data source" that indicates as well if INAV or FNAV (mainly for v3)

For INAV only bits for E1B and E5b can be set (since E5a not transmitted in INAV).

For FNAV only bits for E5a can be set (since E1B & E5b not transmitted in FNAV).